docs/user_guides/fs/feature_group/data_validation.md

```diff
@@ -6,25 +6,29 @@ Clean, high quality feature data is of paramount importance to being able to tra
 
 ## UI
 
-### Step 1: Create or Edit a Feature Group
+### Create a Feature Group (Pre-requisite)
 
-You can attach at most one expectation suite to a Feature Group. It can be done on creation or at a later point in time. Data validation is an optional step and is not required to write to a Feature Group. You can find out more information about creating a Feature Group [here](create.md).
+In the UI, one must create a Feature Group first before attaching an Expectation Suite. You can find out more information about creating a Feature Group [here](create.md). You can attach at most one expectation suite to a Feature Group. Data validation is an optional step and is not required to write to a Feature Group.
```

Comment:
We don't use the "royal we" in docs. It's american style "active voice"

In the UI, you must first create a Feature Group before you can attach an Expectation Suite to it.